.github/workflows/pr.yaml

```diff
@@ -44,6 +44,19 @@ jobs:
           do
             sleep 1
           done
+      - name: Deploy konk-service
+        timeout-minutes: 5
+        run: |
+          make deploy-konk-service
+      - name: Deploy konk-service-operator
+        timeout-minutes: 5
+        run: |
+          make deploy-konk-service-operator
+          make deploy-example-konk-service
+          until kubectl wait --timeout=3m --for=condition=ready pod -l app.kubernetes.io/component=apiservice,app.kubernetes.io/instance=example-konkservice
```

Comment:
This only checks for a pod. In your demo, you actually print CRDs and apiservices. Should we make the e2e test also check for these?

Reply:
Yup, that makes sense.

Reply:
Verified in e2e test - 

```
APIService - 

NAME                                   SERVICE        AVAILABLE                 AGE
v1.                                    Local          True                      3m34s
v1.admissionregistration.k8s.io        Local          True                      3m34s
v1.apiextensions.k8s.io                Local          True                      3m34s
v1.authentication.k8s.io               Local          True                      3m34s
v1.authorization.k8s.io                Local          True                      3m34s
v1.certificates.k8s.io                 Local          True                      3m34s
v1.coordination.k8s.io                 Local          True                      3m34s
v1.events.k8s.io                       Local          True                      3m34s
v1.extensions.example.com              Local          True                      2m51s
v1.rbac.authorization.k8s.io           Local          True                      3m34s
v1.scheduling.k8s.io                   Local          True                      3m34s
v1.stable.example.com                  Local          True                      2m51s
v1alpha1.test.infoblox.com             default/test   False (ServiceNotFound)   2m52s <<<<<<<<
v1beta1.admissionregistration.k8s.io   Local          True                      3m34s
v1beta1.apiextensions.k8s.io           Local          True                      3m34s
v1beta1.authentication.k8s.io          Local          True                      3m34s
v1beta1.authorization.k8s.io           Local          True                      3m34s
v1beta1.certificates.k8s.io            Local          True                      3m34s
v1beta1.coordination.k8s.io            Local          True                      3m34s
v1beta1.discovery.k8s.io               Local          True                      3m34s
v1beta1.events.k8s.io                  Local          True                      3m34s
v1beta1.extensions                     Local          True                      3m34s
v1beta1.node.k8s.io                    Local          True                      3m34s
v1beta1.policy                         Local          True                      3m34s
v1beta1.rbac.authorization.k8s.io      Local          True                      3m34s
v1beta1.scheduling.k8s.io              Local          True                      3m34s

CRDs -

NAME                              CREATED AT
appconfigs.stable.example.com     2020-10-22T03:00:57Z
websites.extensions.example.com   2020-10-22T03:00:57Z
```